### PR TITLE
Octocorp 404

### PR DESCRIPTION
**Pull request by T-Damer**
- Added octo image and some text
- Changed bg to new gradients
- Changed NavBar to octocorp

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -88,6 +88,10 @@ body {
   color: var(--formal-accent);
 }
 
+.stroke-text-secondary {
+  -webkit-text-stroke: 2px var(--secondary);
```
Comment (T-Damer) [inline src/index.css:92]: Supported by 99% of browsers, including FF: https://caniuse.com/?search=-webkit-text-stroke

File: src/components/Navbar/OctoLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const wrapper = classnames(
+  display('flex'),
+  alignItems('items-center'),
+  gap('gap-x-2')
+)
+const logoText = classnames(
+  textColor('text-secondary'),
+  display('hidden', 'sm:block')
+)
+const deadLogo = classnames(
```
Comment (smyzin) [inline src/components/Navbar/OctoLogo.tsx:20]: Interesting class name 🥲

File: src/pages/NotFound.tsx [hunk 3]
```diff
@@ -1,30 +1,76 @@
...
-  const navigate = useNavigate()
+  document.title = '🐙 OCTOCORP'
+
+  useEffect(() => {
+    document.getElementsByTagName('html')[0].setAttribute('bg-theme', '404')
+
+    return () => {
+      document.getElementsByTagName('html')[0].setAttribute('bg-theme', '')
+    }
+  })
```
Comment (smyzin) [inline src/pages/NotFound.tsx:57]: Maybe it's better to move to the hook? It might be useful in the long term

**Review by smyzin — APPROVED** [review]
Looks really good. Thanks

File: src/pages/NotFound.tsx (hunk 3 above)
Comment (T-Damer) [reply src/pages/NotFound.tsx:57]: Let's do this

**Pull request merged**